integrationTest/test/jest-react/.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
I don't think there is a use for this file right?

Reply:
If we ever, for whatever reason, intentionally or unintentionally build the project it could come in handy. It ignores the distributed files.

Reply:
Alright, lets keep it like it is.